libs/utils/src/lib/helpers.ts

```diff
@@ -94,6 +94,10 @@ export async function asyncFilter<T>(items: T[], filterFunction: (item: T) => Pr
   return y.filter(w => w !== _null) as T[];
 }
 
+function findCorrespondence<S extends Scope>(sanitizedCode: string) {
+  return ([key, value]: [string, string]) => [key.toLowerCase(), value.toLowerCase()].includes(sanitizedCode)
+}
```

Comment:
If you've got only two test, use ||. Also you don't use the Scope
```suggestion
function findCorrespondence(code: string) {
  return ([key, value]: [string, string]) => {
    return key.toLowerCase() === code || value.toLowerCase() === code;
   }
}
```